backend/src/middleware/sluggifyMiddleware.js

```diff
@@ -25,9 +25,5 @@ export default {
       args.slug = args.slug || (await uniqueSlug(args.title, isUniqueFor(context, 'Post')))
       return resolve(root, args, context, info)
     },
-    CreateCategory: async (resolve, root, args, context, info) => {
-      args.slug = args.slug || (await uniqueSlug(args.name, isUniqueFor(context, 'Category')))
-      return resolve(root, args, context, info)
-    },
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-11-18T15:11:05Z" title="Monday, November 18th 2019, 4:11:05 pm +01:00">Nov 18, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -25,9 +25,5 @@ export default {
       args.slug = args.slug || (await uniqueSlug(args.title, isUniqueFor(context, 'Post')))
       return resolve(root, args, context, info)
     },
-    CreateCategory: async (resolve, root, args, context, info) => {
-      args.slug = args.slug || (await uniqueSlug(args.name, isUniqueFor(context, 'Category')))
-      return resolve(root, args, context, info)
-    },
```


good cleanup... we shouldn't need to create categories, and if we do, it would be an admin request later